pkgs/by-name/en/en-croissant/package.nix

```diff
@@ -0,0 +1,110 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, stdenvNoCC
+, jq
+, moreutils
+, nodePackages
+, cacert
+, rustPlatform
+, cargo
+, rustc
+, cargo-tauri
+, pkg-config
+, openssl
+, libsoup
+, wrapGAppsHook
+, gtk3
+, webkitgtk
+}:
+
+stdenv.mkDerivation rec {
+  pname = "en-croissant";
+  version = "0.8.0";
+
+  src = fetchFromGitHub {
+    owner = "franciscoBSalgueiro";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-imM7sPV2Yfmi2N/YqOhbiVRBGE/bzuIaqHI0aBHLmTU=";
+  };
+
+  sourceRoot = "${src.name}/src-tauri";
+
+  pnpm-deps = stdenvNoCC.mkDerivation {
+    pname = "${pname}-pnpm-deps";
+    inherit src version;
+
+    nativeBuildInputs = [
+      jq
+      moreutils
+      nodePackages.pnpm
+      cacert
+    ];
+
+    installPhase = ''
+      export HOME=$(mktemp -d)
+      pnpm config set store-dir $out
+      # use --frozen-lockfile to avoid checking git deps
+      pnpm install --frozen-lockfile
```

Comment:
I cant see where the lock file comes from ?

Reply:
This lock file refer to the `pnpm-lock.yaml` that is in the fetched Github repository.

Reply:
i see, `src` gets extracted, I missed that bit